doc/quick_start/getting_started_with_Galactic_Ubuntu20.04_ov2.0.md

```diff
@@ -0,0 +1,169 @@
+sudo python3 downloader.py --name license-plate-recognition-barrier-0001 --output_dir /opt/openvino_toolkit/models/license-plate-recognition/output
+sudo python3 downloader.py --name person-attributes-recognition-crossroad-0230 --output_dir /opt/openvino_toolkit/models/person-attributes/output
+```
+
+* Copy the label file to model output_dir (execute once)
+```
+ sudo cp ~/catkin_ws/src/ros2_openvino_toolkit/data/labels/face_detection/face-detection-adas-0001.labels /opt/openvino_toolkit/models/face_detection/output/intel/face-detection-adas-0001/FP32/
+ sudo cp ~/catkin_ws/src/ros2_openvino_toolkit/data/labels/face_detection/face-detection-adas-0001.labels /opt/openvino_toolkit/models/face_detection/output/intel/face-detection-adas-0001/FP16/
+ sudo cp ~/catkin_ws/src/ros2_openvino_toolkit/data/labels/emotions-recognition/FP32/emotions-recognition-retail-0003.labels /opt/openvino_toolkit/models/emotions-recognition/output/intel/emotions-recognition-retail-0003/FP32/
+ sudo cp ~/catkin_ws/src/ros2_openvino_toolkit/data/labels/object_segmentation/frozen_inference_graph.labels /opt/openvino_toolkit/models/semantic-segmentation/output/FP32/
+ sudo cp ~/catkin_ws/src/ros2_openvino_toolkit/data/labels/object_segmentation/frozen_inference_graph.labels /opt/openvino_toolkit/models/semantic-segmentation/output/FP16/
+ sudo cp ~/catkin_ws/src/ros2_openvino_toolkit/data/labels/object_detection/vehicle-license-plate-detection-barrier-0106.labels /opt/openvino_toolkit/models/vehicle-license-plate-detection/output/intel/vehicle-license-plate-detection-barrier-0106/FP32
+```
+
+* If the model (tensorflow, caffe, MXNet, ONNX, Kaldi) need to be converted to Intermediate Representation (such as the model for object detection):
+  * mobilenet-ssd
+    ```
+    cd ~/openvino/thirdparty/open_model_zoo/tools/model_tools
+    sudo python3 downloader.py --name mobilenet-ssd --output_dir /opt/openvino_toolkit/models/object_detection/mobilenet_ssd/output
+    cd ~/openvino/tools/mo/openvino/tools/mo
+    sudo python3 mo.py --input_model /opt/openvino_toolkit/models/object_detection/mobilenet_ssd/output/public/mobilenet-ssd/mobilenet-ssd.caffemodel --output_dir /opt/openvino_toolkit/models/object_detection/mobilenet_ssd/output
+    ```
+  * deeplabv3
+    ```
+    cd ~/openvino/thirdparty/open_model_zoo/tools/model_tools
+    sudo python3 downloader.py --name deeplabv3 --output_dir /opt/openvino_toolkit/models/deeplabv3/output
+    sudo python3 converter.py --name=deeplabv3 --mo ~/openvino/tools/mo/openvino/tools/mo.py 
+    ```
+
+### Install OpenVINO 2022.1 by PIP
+* OMZ tools are provided for downloading and converting OMZ models in ov2022.([guide](https://pypi.org/project/openvino-dev/))
+
+* See all available models
+```
+omz_downloader --print_all
+```
+
+* Download the optimized Intermediate Representation (IR) of model (execute once), for example:
+```
+omz_downloader --name face-detection-adas-0001 -output_dir /opt/openvino_toolkit/models/face_detection/output
```

Comment:
Hi!
The ''-output_dir"  should be "--output_dir"

Reply:
Fixed.Thanks.